google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -164,8 +164,9 @@
                      key="unique.rest.signature.name" groupKey="inspections.group.name" enabledByDefault="true"
                      level="ERROR" implementationClass="com.google.cloud.tools.intellij.appengine.validation.RestSignatureInspection"/>
 
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.PluginCompatibilityCheck"/>
-    <postStartupActivity implementation="com.google.cloud.tools.intellij.ConflictingAppEnginePluginCheck"/>
+    <postStartupActivity implementation="com.google.cloud.tools.intellij.startup.PluginCompatibilityCheck"/>
```

Comment:
One thing to keep in mind (I'm coming in late to this PR) is that you should probably check for the existence of App Engine framework support / AE facet for this check rather than manually reading the appengine-web.xml again (since it has already been read by the time the framework support has been added).